client/state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js

```diff
@@ -108,41 +109,37 @@ const trackEligibility = data => {
 };
 
 /**
- * Dispatches to update eligibility information from API response
+ * Issues an API request to fetch eligibility information for a site
  *
- * @param {Function} dispatch action dispatcher
- * @param {number} siteId site for which the update belongs
- * @returns {Function} the handler function with site and dispatch partially applied
+ * @param {Object} store Redux store
+ * @param {Function} store.dispatch action dispatcher
+ * @param {action} action Action object
  */
-const apiResponse = ( dispatch, siteId ) => data => {
+export const requestEligibility = ( { dispatch }, action ) => {
+	const { siteId } = action;
+
+	dispatch( http( {
+		method: 'GET',
+		path: `/sites/${ siteId }/automated-transfers/eligibility`,
+		apiVersion: '1',
+	}, action ) );
+};
+
+export const receiveResponse = ( { dispatch }, { siteId }, data ) => {
```

Comment:
Optional, but we could make the function name a bit more descriptive. For example maybe `updateAutomatedTransferEligibility` over names like `handleSuccess` or `onClick`.

Reply:
Yeah, that is a good suggestion. I'll adjust.